build.gradle

```diff
@@ -59,8 +59,9 @@ dependencies {
     compile("com.jayway.jsonpath:json-path:$jsonPathVersion")
 
     compile('org.springframework.boot:spring-boot-starter-validation')
-    compile('org.springframework.boot:spring-boot-starter-actuator')
     compile('org.springframework.boot:spring-boot-starter-web')
+    compile('org.springframework.boot:spring-boot-starter-actuator')
+    compile "io.micrometer:micrometer-registry-prometheus:latest.release"
```

Comment:
it is better to use a fixed version I believe

Reply:
done